public/i18n/en.json

```diff
@@ -12,7 +12,7 @@
   "ftm-accepting": "Accepting",
   "ftm-accepts-open-boxes": "Accepts Opened Boxes/bags",
   "ftm-ack-data-sources": "This site also includes data from <a href=\"https://tinyurl.com/hospitals-needing-ppe\">https://tinyurl.com/hospitals-needing-ppe</a>, a project by:",
-  "ftm-add-donation-site-answer": "<a class='add-donation-site-form'><button style='margin-top:20px;' type='button' class='btn-sm btn-primary'>Complete this form</button></a>",
+  "ftm-add-donation-site-answer": "<a class='add-donation-site-form' target='_blank'><button style='margin-top:20px;' type='button' class='btn-sm btn-primary'>Complete this form</button></a>",
```

Comment:
can you also add `rel="noopener noreferrer"` since you're adding `target="_blank"`?

https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
Is it possible to move the `<a>` and `<button>` into the `.handlebars` file? That would make this easier to maintain and less prone to inconsistencies across locales.

Reply:
ya that would be better, I think there used to be text before the `<a>` that was removed recently.

Reply:
@saikatbhadra ping ^^

Reply:
sorry i forgot about this, how do I do variable substitution with i18n